migrate/20240503_boot_image.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:boot_image) do
+      column :id, :uuid, primary_key: true
+      foreign_key :vm_host_id, :vm_host, type: :uuid, null: false
+      column :name, :text, collate: '"C"', null: false
+      column :version, :text, collate: '"C"', null: false
+      column :active, :boolean, null: false, default: false
```

Comment:
What do you think about converting this column a timestamp as `activated_at`? It could be useful to know when we activated this version on the host. `nil` can indicate not activated.